sqllineage/__init__.py

```diff
@@ -74,4 +74,6 @@ def _monkey_patch() -> None:
     "SQLLINEAGE_DIRECTORY", os.path.join(os.path.dirname(__file__), "data")
 )
 DEFAULT_HOST = "localhost"
-DEFAULT_PORT = 5000
+DEFAULT_PORT = 5001
```

Comment:
We already have an option --port to let user specify PORT used for web UI, let's not change the default port here.